surrealdb/clients/ws.py

```diff
@@ -203,13 +237,10 @@ async def signin(
 
         if username is not None:
             self._username = username
-
         if password is not None:
             self._password = password
-
         if namespace is not None:
             self._namespace = namespace
-
```

Comment:
why are we removing these whitespaces? it separates out the logic and removing them is harder to read